core/templates/pages/exploration-player-page/layout-directives/learner-local-nav.component.html

```diff
@@ -27,13 +27,11 @@
     </a>
   </li>
   <li class="nav-item">
-    <a *ngIf="username" (click)="showFlagExplorationModal()" href="#"
-       (keydown.enter)="showFlagExplorationModal()"
-       ngbTooltip="{{ 'I18N_PLAYER_REPORT_TOOLTIP' | translate }}" placement="bottom"
-       tabindex="0" class="nav-link e2e-test-report-exploration-button">
+    (keydown.enter)="showFlagExplorationModal()"    <div tabindex="0" *ngIf="username" class="nav-link e2e-test-report-exploration-button"
```

Comment:
` (keydown.enter)="showFlagExplorationModal()"`, is outside of any tag and is repeated in the div tag that follows, please remove.

Reply:
@Lawful2002 thanks for pointing! I have fixed that